packages/@adobe/spectrum-css-temp/components/button/skin.css

```diff
@@ -394,7 +394,7 @@ governing permissions and limitations under the License.
   &:disabled,
   &.is-disabled {
     background-color: var(--spectrum-actionbutton-background-color-disabled);
-    border-color: var(--spectrum-actionbutton-border-color-disabled);
+    border-color: var(--spectrum-actionbutton-border-color);
```

Comment:
Tested in the storybook(overflow mode; collapse; disabled keys (changed renderCollapsible function in actiongroup story to overflow: wrap for testing)), and the border is showing up. 

Looked for other references of the same variable and did not find any.

Reply:
I think this will cause the entire button to get a non-disabled border, we only want it on the side that is clearly different from the others
See the image from the Issue description here, with highlighting of the edge missing its border
![143933078-a5887883-0078-4588-ba2f-c62114dc06d0](https://user-images.githubusercontent.com/698229/145127307-68daf3d3-6cac-482b-ba92-ef3ca110e8f1.png)

you may find it useful to use `border-inline-start-color` and we'll probably need another story with a button in the middle of a selected group that is disabled, these are both on the edges

Reply:
btw, this is what this PR generated
![Screen Shot 2021-12-07 at 4 36 52 PM](https://user-images.githubusercontent.com/698229/145127532-8e4e61fd-dc3b-4d2a-ad4c-c8a2e3990176.png)
you can see the extra border all around Edit and Duplicate when we only want it between Move and Duplicate

Reply:
yes, I do see the extra border. Thanks for taking a look and all the details. I will try using "border-inline-start-color"

Reply:
I tried "border-inline-start-color"  but the color takes effect for all disabled-buttons. Searching for alternatives.

Reply:
can we consider a border color with less opacity?

Reply:
sorry, i missed your replies. opacity won't be right. what you may need to do is introduce a new class on buttons whose are disabled, but whose previous sibling wasn't. For those, use border-inline-start